### DISCUSSION TITLE
Solution resolved found in the documents. Thank you.

### DISCUSSION THREAD

**Discussion by hugolimav8** [devlikeapro/waha#348] [General]
[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by allburov** [comment]
:+1: Could you share may be the question and found solution to help the future people? 

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
